### PR TITLE
community/py3-texttable: upgrade to 1.6.2

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: community/py3-texttable/APKBUILD [hunk 1]
```diff
@@ -1,29 +1,26 @@
...
+pkgrel=0
 pkgdesc="module for creating simple ASCII tables"
 url="https://github.com/foutaise/texttable/"
-arch="all"
-license="LGPL-3.0"
+arch="noarch"
+license="MIT"
 depends="python3"
-makedepends="python3-dev"
+makedepends="py3-setuptools"
```
Comment (prspkt) [inline community/py3-texttable/APKBUILD]:
```suggestion
makedepends="py3-setuptools"
checkdepends="py3-pytest"
```

File: community/py3-texttable/APKBUILD [hunk 2]
```diff
@@ -1,29 +1,26 @@
...
 builddir="$srcdir/texttable-$pkgver"
 
 build() {
-	cd "$builddir"
 	python3 setup.py build
 }
 
 check() {
-	cd "$builddir"
 	python3 setup.py check
```
Comment (prspkt) [inline community/py3-texttable/APKBUILD]:
```suggestion
	py.test-3 tests.py -v
```

Comment (maxice8) [reply community/py3-texttable/APKBUILD]: fixed

File: community/py3-texttable/APKBUILD (hunk 1 above)
Comment (maxice8) [reply community/py3-texttable/APKBUILD]: fixed

**Pull request closed without merging**